build_docs_with_style.jl

```diff
@@ -1,29 +1,37 @@
 CI = get(ENV, "CI", nothing) == "true" || get(ENV, "GITHUB_TOKEN", nothing) !== nothing
 
 import Pkg
-Pkg.pkg"add Documenter@0.27"
+Pkg.pkg"add Documenter@1"
```

Comment:
We don't need this at all now, latest docuenter will be installed by defalt.

Reply:
Perhaps we should specify Documenter v1 in Project.toml instead, because there is breaking syntax in this PR? This avoids potentially messing stuff up if running this locally

Reply:
No, this would require you to alter the project.toml in 15+ packages. We can laeve it as is for now, as this particualr script only works for version 1 anyways.